.github/workflows/pull_request_rust.yml

```diff
@@ -0,0 +1,28 @@
+name: Pull Request (Rust)
+
+on:
+  pull_request:
```

Comment:
Will this be re-run for every push to a PR,  or just run once when the PR is created?

Reply:
all pushes to a pr, it seems

Reply:
👍